apps/taskman/api.py

```diff
@@ -223,3 +223,29 @@ def put(self, request, task_key):
         ).update_task_status(
             issue_key=task_key, status=serializer.validated_data["status"]
         )
+
+
+@jira_token_description
+class task_assignee(GenericAPIView):
```

Comment:
why are class names snake case?

Reply:
both `collectors/framework/api.py` and `apps/osim/api.py` are snake case, I though it would be the guideline for our API classes, but I'm up to change if you prefer.

Reply:
eh whatever, I'd prefer to stick to Python's conventions (which we do in OSIDB's api views) but unless we change all of them it doesn't change much here... odd that the linters don't pick this up